fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/FDBLuceneQueryTest.java

```diff
@@ -514,19 +518,19 @@ public void delayFetchOnOrOfLuceneFiltersGivesUnion(boolean shouldDeferFetch) th
             Matcher<RecordQueryPlan> matcher = union(
                     indexScan(allOf(indexScan("Complex$text_index"),
                             indexScanType(IndexScanType.BY_LUCENE_FULL_TEXT),
-                            bounds(hasTupleString("[[(civil blood makes civil hands unclean)],[(civil blood makes civil hands unclean)]]")))),
+                            bounds(hasTupleString("[[(\"civil blood makes civil hands unclean\")],[(\"civil blood makes civil hands unclean\")]]")))),
```

Comment:
Does this imply that the bounds to scans of the `LuceneIndexMaintainer` need to change? I believe that's tested in the `LuceneIndexTest`, so it may not, but if users have to start supplying quotes to their tuple bounds, that's something we should at least note somewhere as something they should react to

Reply:
Added a note saying the bounds are Lucene parsable strings...  I think the problem is that (_text1 text2_) is not the same as ("_text1 text2_") in the syntax specification.